src/components/QaList/QaList.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react'
+import * as Styled from './styled'
+import 'dayjs/locale/ja'
+import {
+  Event,
+  Talk,
+  useGetApiV1ChatMessagesQuery,
+} from '../../generated/dreamkast-api.generated'
+import { setupDayjs } from '../../util/setupDayjs'
+import { CardContent, Grid } from '@material-ui/core'
+import { Chat } from '../Chat'
+import dayjs from 'dayjs'
+import { settingsSelector } from '../../store/settings'
+import { useSelector } from 'react-redux'
+
+setupDayjs()
+
+type Props = {
+  event: Event
+  talks: Talk[]
+}
+
+export const QaList: React.FC<Props> = ({ event, talks }) => {
+  const settings = useSelector(settingsSelector)
+  const [talk, setTalk] = useState<Talk | undefined>(undefined)
+  const { data, isLoading, isError, error } = useGetApiV1ChatMessagesQuery({
+    eventAbbr: event.abbr,
+    roomType: 'talk',
+    profileId: `${settings.profile.id}`,
+  })
```

Comment:
```suggestion
  },
  { skip: !settings.profile.id },  
)
```

skipを入れないと、profileの取得が完了するまでに `0` か `null` か何かしらのfalsy valueでAPI callしてしまいます。